c2corg_ui/static/partials/trackdownload.html

```diff
@@ -0,0 +1,2 @@
+<button class="btn btn-primary" ng-click="trackCtrl.downloadGpx()">GPX</button>
```

Comment:
I was going to make the same comment. The button is too big IMHO, a link would be more consistent with the content around.